packaging/nuget/nservicebus.ravendb.nuspec

```diff
@@ -16,7 +16,7 @@
     <tags>nservicebus servicebus msmq cqrs publish subscribe ravendb</tags>
     <dependencies>
       <dependency id="NServiceBus" version="[5.2.8, 6.0.0)" />
-      <dependency id="RavenDB.Client" version="[3.0.3800, 4.0.0)" />
+      <dependency id="RavenDB.Client" version="[3.0.3800, 3.1.0)" />
```

Comment:
@SimonCropp as @DavidBoike pointed out, and details are in #300, it is broken now. Meaning that users can't upgrade to RavenDB.Client 3.5 due to a breaking change in RavenDB. There is really nothing we can do other that trying to prevent that.